install/tools/setupMeshEx.sh

```diff
@@ -84,11 +83,12 @@ function istioDnsmasq() {
 # Parameters:
 # - name of the k8s cluster.
 function istioClusterEnv() {
-   local K8S_CLUSTER=${1:-${K8S_CLUSTER}}
+  local K8S_CLUSTER=${1:-${K8S_CLUSTER}}
+  K8S_GCP_OPTS=${K8S_GCP_OPTS:-GCP_OPTS}
```

Comment:
why is this a different set of options ?

Reply:
I see a few of your comments are related to this so I will just reply once here. 
The cluster and the vm might be in different zone. By having two set of options (one for cluster and one for vm, both default to GCP_OPTS), it is easier to specify the zone using the same set of env var exports.

Reply:
you could also change it between invocation ie
```
GCP_OPTS="--zone A" setupMeshEx.sh ...
GCP_OPTS="--zone B" setupIstioVM.sh ...
```